docs/kratos/guides/digital-ocean-nginx-systemd.mdx

```diff
@@ -0,0 +1,580 @@
+
+
+- **OS**: Ubuntu 20.04
+- **Plan**: Basic
+- **CPU options**: Regular with SSD
+- **RAM**: 1Gb
+- **SSD**: 25Gb
+- **VPC network**: default
+- **Authentication**: SSH Keys. Don't forget to add your own SSH key
+- **Region**: Choose your region
+
+Wait for the ready-to-use VM and copy the IP address of your droplet
+
+:::note
+This example shows a single configuration on a small droplet.
+The configuration of a droplet may vary depending on your scale
+:::
+
+This guide uses accounts.example.com as a hostname to run Kratos.
+You need to configure DNS, create an `A type` record, and point it to the droplet's IP.
+
+Let's connect to our droplet via SSH
+
+```bash
+ssh -A root@accounts.example.com
+The authenticity of host 'accounts.example.com (157.245.71.211)' can't be established.
+ED25519 key fingerprint is SHA256:WJsAOHk3b12Ym7keL8MHjBLjgvxfUQPhIy9pwx1czAk.
+This key is not known by any other names
+Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
+Warning: Permanently added 'accounts.example.com' (ED25519) to the list of known hosts.
+Welcome to Ubuntu 20.04.3 LTS (GNU/Linux 5.4.0-97-generic x86_64)
+ * Documentation:  https://help.ubuntu.com
+ * Management:     https://landscape.canonical.com
+ * Support:        https://ubuntu.com/advantage
+
+  System information as of Fri Apr 15 07:57:35 UTC 2022
+
+  System load:  0.31              Users logged in:       0
+  Usage of /:   6.1% of 24.06GB   IPv4 address for eth0: 157.245.71.211
+  Memory usage: 19%               IPv4 address for eth0: 10.18.0.5
+  Swap usage:   0%                IPv4 address for eth1: 10.110.0.2
+  Processes:    109
+
+1 update can be applied immediately.
+To see these additional updates run: apt list --upgradable
+
+
+
+The programs included with the Ubuntu system are free software;
+the exact distribution terms for each program are described in the
+individual files in /usr/share/doc/*/copyright.
+
+Ubuntu comes with ABSOLUTELY NO WARRANTY, to the extent permitted by
+applicable law.
+
+root@ubuntu-s-1vcpu-1gb-ams3-01:~#
+```
+
+## Installing required dependencies
+As a first step, we need to upgrade all packages in our Droplet.
+
+```bash
+apt-get update && apt-get upgrade
+```
+The default version of nodejs is v10.19.0 on Ubuntu 20.04,
+which seems outdated for today. We need to install a newer version.
+
+```bash
+curl -sL https://deb.nodesource.com/setup_16.x -o /tmp/nodesource_setup.sh
+bash /tmp/nodesource_setup.sh
+apt-get install nodejs npm jq unzip
+```
+
+## Installing PostgreSQL
+
+Let's install PostgreSQL by running the following command
+
+```bash
+sudo apt install postgresql postgresql-contrib
+sudo -i -u postgres
+```
+Let's create database
+
+```bash
+createdb kratos
+```
+
+Let's change the default password encryption to a stronger one
+
+```bash
+psql
+ALTER SYSTEM SET password_encryption = 'scram-sha-256'; # Change the default password encryption to stronger one
+ALTER SYSTEM
+
+SELECT pg_reload_conf();
+```
```

Comment:
This fails for me with:


```
postgres@ubuntu-s-1vcpu-1gb-fra1-01:~$ psql psql (12.10 (Ubuntu 12.10-0ubuntu0.20.04.1)) Type "help" for help. postgres=# ALTER SYSTEM SET password_encryption = 'scram-sha-256'; # Change the default password encryption to stronger one ALTER SYSTEM postgres-# ALTER SYSTEM postgres-# postgres-# SELECT pg_reload_conf(); ERROR: syntax error at or near "#" LINE 1: # Change the default password encryption to stronger one
```

This fixes the comment:

````suggestion

```bash
psql
-- Change the default password encryption to stronger one
ALTER SYSTEM SET password_encryption = 'scram-sha-256'; 
ALTER SYSTEM

SELECT pg_reload_conf();
```
````

However, it still fails:

```
postgres@ubuntu-s-1vcpu-1gb-fra1-01:~$ psql
psql (12.10 (Ubuntu 12.10-0ubuntu0.20.04.1))
Type "help" for help.

postgres=# -- Change the default password encryption to stronger one
postgres=# ALTER SYSTEM SET password_encryption = 'scram-sha-256'; 
ALTER SYSTEM
postgres=# ALTER SYSTEM
postgres-# 
postgres-# SELECT pg_reload_conf();
ERROR:  syntax error at or near "SELECT"
LINE 2: SELECT pg_reload_conf();
```